docusaurus.config.ts

```diff
@@ -73,29 +74,37 @@ const config: Config = {
 			style: "dark",
 			links: [
 				{
-					title: "Docs",
+					title: "Resources",
 					items: [
 						{
-							label: "Docs",
+							label: "Documentation",
 							to: "/docs/installation",
 						},
 						{
-							label: "Showcase",
-							to: "/showcase",
+							label: "Features",
+							to: "/features",
+						},
+						{
+							label: "Blog",
+							to: "/blog",
 						},
 					],
 				},
 				{
 					title: "Community",
 					items: [
 						{
-							label: "Discord (English mainly)",
```

Comment:
I prefer the old one, which can provide convenience for users who only speak English/Chinese but don't have Discord/Telegram.

Reply:
Hmm alright.